src/state/MuteAllAudioModel.ts

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE in the repository root for full details.
+*/
+
+import { combineLatest, map, startWith } from "rxjs";
+
+import { setOutputEnabled$ } from "../controls";
+import { muteAllAudio as muteAllAudioSetting } from "../settings/settings";
+
+/**
+ * This can transition into sth more complete: `GroupCallViewModel.ts`
+ */
+export const muteAllAudio$ = combineLatest([
+  setOutputEnabled$,
+  muteAllAudioSetting.value$,
+]).pipe(
+  startWith([true, muteAllAudioSetting.getValue()]),
+  map(([outputEnabled, settingsMute]) => !outputEnabled || settingsMute),
+);
```

Comment:
```suggestion
export const muteAllAudio$ = combineLatest([
  setOutputEnabled$.pipe(startWith(true)),
  muteAllAudioSetting.value$,
], (outputEnabled, settingsMute) => !outputEnabled || settingsMute);
```

Since the setting is already behavior-like